app/queries/get_change_offer_options.rb

```diff
@@ -44,10 +44,8 @@ def available_sites(course:, study_mode:)
 
   def offerable_courses
     make_decisions_courses
-      .where(
-        open_on_apply: true,
-        recruitment_cycle_year: recruitment_cycle_year,
-      )
+      .where(recruitment_cycle_year: recruitment_cycle_year)
+      .open_on_apply
```

Comment:
nit: can we move this before the `.where`?
Also can you add a test to check that we only include courses which are both `open_on_apply` and exposed_in_find`?

Reply:
We already have a test checking whether `open_on_apply` is taken into account, maybe it's worth modifying this to manually set `open_on_apply: true` for the excluded course, as the `:open_on_apply` factory trait also sets `exposed_in_find`.

Reply:
Done, I've amended an existing test to make it better